From c37a6401469cae16519db06d623d49c6f606f312 Mon Sep 17 00:00:00 2001
From: Thomas Lamprecht <t.lamprecht@proxmox.com>
Date: Wed, 31 Oct 2018 15:37:39 +0100
Subject: [PATCH] Revert "Install init scripts to support non-systemd setups.
 (Closes: #826994)"

This reverts commit 2ad6c7f9e150e9c2fa146891785b09465aa0c7e5.

Breaks on systems with sysv-rc installed (possible on Jessie upgraded
systems)

Signed-off-by: Thomas Lamprecht <t.lamprecht@proxmox.com>
---
 debian/rules                                 | 8 --------
 debian/zfs-zed.zfs-zed.init                  | 1 -
 debian/zfsutils-linux.zfs-import.init        | 1 -
 debian/zfsutils-linux.zfs-mount.init         | 1 -
 debian/zfsutils-linux.zfs-share.init         | 1 -
 5 files changed, 12 deletions(-)
 delete mode 120000 debian/zfs-zed.zfs-zed.init
 delete mode 120000 debian/zfsutils-linux.zfs-import.init
 delete mode 120000 debian/zfsutils-linux.zfs-mount.init
 delete mode 120000 debian/zfsutils-linux.zfs-share.init

diff --git a/debian/rules b/debian/rules
index 5fba58ff..81c301e4 100644
--- a/debian/rules
+++ b/debian/rules
@@ -161,14 +153,6 @@ override_dh_install:
 	find . -name lib*.la -delete
 	dh_install --fail-missing
 
-override_dh_installinit:
-	dh_installinit -r --no-start --name zfs-import
-	dh_installinit -r --no-start --name zfs-mount
-	dh_installinit -r --no-start --name zfs-share
-	dh_installinit -R --no-start --name zfs-zed
-	mkdir -p debian/zfsutils-linux/lib/systemd/system
-	ln -sr /dev/null debian/zfsutils-linux/lib/systemd/system/zfs-import.service
-
 # ------------
 
 override_dh_prep-deb-files:
diff --git a/debian/zfs-zed.zfs-zed.init b/debian/zfs-zed.zfs-zed.init
deleted file mode 120000
index 3f41f681..00000000
--- a/debian/zfs-zed.zfs-zed.init
+++ /dev/null
@@ -1 +0,0 @@
-../etc/init.d/zfs-zed
\ No newline at end of file
diff --git a/debian/zfsutils-linux.zfs-import.init b/debian/zfsutils-linux.zfs-import.init
deleted file mode 120000
index bfc368fd..00000000
--- a/debian/zfsutils-linux.zfs-import.init
+++ /dev/null
@@ -1 +0,0 @@
-../etc/init.d/zfs-import
\ No newline at end of file
diff --git a/debian/zfsutils-linux.zfs-mount.init b/debian/zfsutils-linux.zfs-mount.init
deleted file mode 120000
index 62a54433..00000000
--- a/debian/zfsutils-linux.zfs-mount.init
+++ /dev/null
@@ -1 +0,0 @@
-../etc/init.d/zfs-mount
\ No newline at end of file
diff --git a/debian/zfsutils-linux.zfs-share.init b/debian/zfsutils-linux.zfs-share.init
deleted file mode 120000
index 3f069f9b..00000000
--- a/debian/zfsutils-linux.zfs-share.init
+++ /dev/null
@@ -1 +0,0 @@
-../etc/init.d/zfs-share
\ No newline at end of file
-- 
2.19.1

